README.md

```diff
@@ -39,6 +39,36 @@ Brand new or update to a contribution via these methods:
 * After submission, check the [Pull Request](https://github.com/Azure/Azure-Sentinel/pulls) for comments
 * Make changes as suggested and update your branch or explain why no change is needed. Resolve the comment when done.
 
+### Pull Request Kql Validation Check
+As part of the PR checks we run kql validation on the queries that are defined in the template. If this check fails go to Azure Pipeline (by pressing on the errors link on the checks tab in your PR)
+![Azurepipeline](.github/Media/Azurepipeline.png)
+In the pipeline you can see which test fail and what is the cause:
+![Pipeline Tests Tab](.github/Media/PipelineTestsTab.png)
+If you are using custom logs table (a table which is not defined on all workspaces by default) you should verify
```

Comment:
nit: a custom logs table
Just to be super accurate, I would say "a table that ends with _CL" (instead of not defined by default). This is how a custom log table is officially defined:
https://docs.microsoft.com/en-us/azure/azure-monitor/platform/data-sources-custom-logs#step-4-provide-a-name-and-description-for-the-log

Reply:
Oh, now i see that it's not only custom log tables, but any table that is not defined in our nugget.
In that case I would either list the tables that are defined or not say that it's a custom logs table but any table that is not already defined either in the nugget or in this repo

Reply:
let's talk offline on it @mgladi